projects/m4/011-distinct-names/js/index.js

```diff
@@ -0,0 +1,62 @@
+const fs = require("fs");
+const fsPromises = fs.promises;
+const path = require("path");
+const data = path.join(__dirname, "data");
+
+// MAIN FUNCTION
+// Read all the files from a directory and print all the unique popular names (female and male)
+fs.readdir(data, (err, files) => {
+  createObjectNames(files).then((object) => {
+    getNames(object.femaleNames).then((names) =>
+      console.log("Popular female names", names)
+    );
+    getNames(object.maleNames).then((names) =>
+      console.log("Popular male names", names)
+    );
```

Comment:
Does the function also handle any **errors**? 
I don't see "`catch`" here and in the function `getNames` ;-)